examples/flutter_gallery/lib/demo/material/list_demo.dart

```diff
@@ -64,8 +64,8 @@ class _ListDemoState extends State<ListDemo> {
   }
 
   void _showConfigurationSheet() {
-    final PersistentBottomSheetController<void> bottomSheet =
-        scaffoldKey.currentState.showBottomSheet<void>((BuildContext bottomSheetContext) {
+    final PersistentBottomSheetController<void> bottomSheet = scaffoldKey.currentState
+      .showBottomSheet<void>((BuildContext bottomSheetContext) {
```

Comment:
Indentation to remind the reader that we're within the function's body needs to be included here:
```
      .showBottomSheet<void>((BuildContext bottomSheetContext) {
        return Container(
          decoration: const BoxDecoration(
            border: Border(top: BorderSide(color: Colors.black26)),
          ),
          ...
       });
```

Reply:
That's pretty bad.

Here's how I would have formatted it:
```
    final PersistentBottomSheetController<void> bottomSheet = scaffoldKey.currentState
      .showBottomSheet<void>((BuildContext bottomSheetContext) {
        return Container(
          decoration: const BoxDecoration(
            border: Border(top: BorderSide(color: Colors.black26))
          ),
        );
      });
```

As I've said before, if there was a way for the formatter to recognize the above as a legitimate (alternative) way to format the code and leave it alone, then all would be well.